src/core/frameworks/gql/queries.ts

```diff
@@ -314,45 +310,104 @@ export const GET_BORROWER_POSITIONS_QUERY = gql`
   ${SPIGOT_EVENT_FRAGMENT}
   ${ESCROW_FRAGMENT}
 
-  query getBorrowerPositions($borrower: String!) {
-    lineOfCredits(where: { borrower_contains: $borrower }) {
-      ...BaseLineFrag
+  fragment PositionsFrag on LineOfCredit {
+    ...BaseLineFrag
 
-      positions(first: 20) {
-        ...LinePageCreditFrag
+    positions {
+      ...LinePageCreditFrag
+    }
+
+    events {
+      ...LineEventFrag
+    }
+
+    spigot {
+      id
+      spigots {
+        ...BaseSpigotFrag
       }
 
-      events(first: 20) {
-        ...LineEventFrag
+      summaries {
+        ...SpigotSummaryFrag
       }
+      events {
+        ...SpigotEventFrag
+      }
+    }
+    escrow {
+      ...EscrowFrag
+    }
+  }
+`;
 
-      spigot {
+// fetches all of a users positions that they lend to
+const LENDER_POSITIONS_FRAGMENT = gql`
+  ${BASE_LINE_FRAGMENT}
+  ${LINE_EVENT_FRAGMENT}
+  ${BASE_SPIGOT_FRAGMENT}
+  ${SPIGOT_SUMMARY_FRAGMENT}
+  ${SPIGOT_EVENT_FRAGMENT}
+  ${ESCROW_FRAGMENT}
+
+  fragment LenderPositionsFrag on Lender {
+    positions: positions {
```

Comment:
isnt this repeating the same data fields above? Why not use the fragment u just created?

Reply:
I dont really see the difference between borrower vs lender vs linepage queries. Line Page already has almost all data available, we do we need multiple identical queries?

Reply:
> Line Page already has almost all data available, we do we need multiple identical queries?

I did not dig deeply into `GET_LINE_PAGE_QUERY` (e.g. `getLinePage($id: ID!)`) and `GET_LINE_PAGE_AUX_QUERY` (e.g. `getLinePageAux($id: ID)`), but it should not be noted that these queries get data for only a SINGLE line of credit, not all lines of credit for a given borrower/lender/arbiter which is the intention of this PR. 

As a reminder, the `GET_USER_PORTFOLIO_QUERY` (e.g. `getUserPortfolio($user: String!)`) replaces and extends the previously merged in `GET_BORROWER_POSITIONS_QUERY` (e.g. `getBorrowerPositions($borrower: String!)`) to include `lenderPositions` and `arbiterLineOfCredits`. 

I don't think we have multiple identical queries, and I don't think it's worth the time/energy to simplify them further (but let me know if you disagree!).


> I dont really see the difference between borrower vs lender vs linepage queries.

Borrower (or Arbiter) and Lender have different queries because they have different underlying data structures. For Borrower and Arbiter, you query their `LineOfCredits` and get the `positions` for each `LineOfCredit`; `Lenders` don't have `LinesOfCredit`, just positions, so you directly query for the `positions`. This is why there are separate fragments for Borrowers/Arbiters, `POSITIONS_FRAGMENT` (renamed to `LINE_OF_CREDITS_FRAGMENT`), and Lenders, `LENDER_POSITIONS_FRAGMENT`.